R/model-tools.R

```diff
@@ -302,44 +307,43 @@ enw_sample <- function(data, model = epinowcast::enw_model(),
 #' @importFrom cmdstanr cmdstan_model
 #' @examplesIf interactive()
 #' mod <- enw_model()
-enw_model <- function(model, include, compile = TRUE,
-                      threads = FALSE, profile = FALSE,
-                      stanc_options = list(), verbose = TRUE, ...) {
-  if (missing(model)) {
-    model <- "stan/epinowcast.stan"
-    model <- system.file(model, package = "epinowcast")
-  }
-  if (missing(include)) {
-    include <- system.file("stan", package = "epinowcast")
+enw_model <- function(
+    model = system.file("stan", "epinowcast.stan", package = "epinowcast"),
+    include = dirname(model),
```

Comment:
This changes default behaviour with a user-specified model if specifying here then it would be better to specify directly vs dependent on `model` (as the simplest use case is making a slight change in `epinowcast.stan` but wanting all of our functions.

Reply:
fair enough - will make that tweak.